components/datatypes/formapplication.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/formapplication",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Form Application Details",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Used when any application process is started, such as financing, limit changes, or educational programs.",
+  "definitions": {
+    "formApplicationDetails": {
+      "properties": {
+        "xdm:formApplicationID": {
+          "title": "Form Application ID",
+          "type": "string",
+          "description": "Identifier used during an application process."
+        },
+        "xdm:FormApplicationType": {
+          "title": "Form Application Type",
+          "type": "string",
+          "description": "Type of the application process."
+        },
+        "xdm:formApplicationUsage": {
+          "title": "Form Application Usage",
+          "type": "data type: toolusage",
+          "description": "Extension of toolUsage data type."
+        },
+        "xdm:formApplicationApproved": {
+          "title": "Form Application Approved",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Captured when a visitor application has been approved."
+        },
+        "xdm:formApplicationDeclined": {
+          "title": "Form Application Declined",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Captured when a visitor application has been declined."
+        },
+        "xdm:formApplicationUnderReview": {
+          "title": "Form Application Under Review",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Captured when a visitor application has been placed under review."
+        },
+        "xdm:formApplicationStatus": {
+          "title": "Form Application Status",
+          "enum":["initiated", "in progress", "approved", "declined"],

```

Comment:
All enums should have corresponding meta:enums defined. Please refer schemas in GITHUB for examples.